SparkPost/2web2ui
# AC-1083 - "Show Me SparkPost" checklist item for sending test email

AC-1083 - "Show Me SparkPost" checklist item for sending test email

### What Changed
 - "Show Me SparkPost" Button directs to the Templates page / starts the guide in AC-1107
 - Condition is stored in UI option on clicking the button and navigating to the Templates page. (The ListItem will have a green tick and button is no longer orange)

### How To Test
 -  Click on 'Start Sending' then 'Show Me SparkPost' then 'Send a Test Email'.
 -  Check if click on Send a Test Email button redirects to /templates page, check after redirection 
      the list item is completed(green tick and button is no longer orange. )

### To Do
